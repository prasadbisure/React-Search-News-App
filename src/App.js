import './App.css';
import Landing from './pages/Landing'
import SearchResult from './pages/SearchResult';
import {Route, Switch, BrowserRouter} from 'react-router-dom';

const App= () => {

   return (
    <div className="App">
      <BrowserRouter>
        <Switch>
          <Route exact path="/search" component={SearchResult}/>
          <Route path="/" component={Landing}/>
        </Switch>
      </BrowserRouter>
    </div>
  );
}

export default App;
