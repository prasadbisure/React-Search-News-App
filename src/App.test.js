import { shallow } from 'enzyme';
import App from './App';

describe('SearchKeyword component', () => {
    it('should render snapshots properly', () =>{
        const wrapper =  shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
})
