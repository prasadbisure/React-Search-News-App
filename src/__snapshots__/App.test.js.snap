// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchKeyword component should render snapshots properly 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/search"
      />
      <Route
        component={[Function]}
        path="/"
      />
    </Switch>
  </BrowserRouter>
</div>
`;
